homeassistant/components/sensor/pollen.py

```diff
@@ -105,69 +83,71 @@
     'maximum': 12
 }]
 
+TREND_FLAT = 'Flat'
+TREND_INCREASING = 'Increasing'
+TREND_SUBSIDING = 'Subsiding'
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_ZIP_CODE): str,
-    vol.Required(CONF_MONITORED_CONDITIONS):
-        vol.All(cv.ensure_list, [vol.In(CONDITIONS)]),
+    vol.Required(CONF_MONITORED_CONDITIONS, default=list(SENSORS)):
+        vol.All(cv.ensure_list, [vol.In(SENSORS)]),
+    vol.Optional(CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL):
```

Comment:
Remove this. Every platform on home-assistant support this by default. Use Throttle to protect the API but that need not be configurable.